apps/webapp/app/v3/authenticatedSocketConnection.server.ts

```diff
@@ -50,8 +53,42 @@ export class AuthenticatedSocketConnection {
     ws.addEventListener("close", this.#handleClose.bind(this));
     ws.addEventListener("error", this.#handleError.bind(this));
 
+    ws.on("ping", (data) => {
+      logger.debug("[AuthenticatedSocketConnection] Received ping", {
+        id: this.id,
+        envId: this.authenticatedEnv.id,
+        data,
+      });
+    });
+
+    ws.on("pong", (data) => {
+      logger.debug("[AuthenticatedSocketConnection] Received pong", {
+        id: this.id,
+        envId: this.authenticatedEnv.id,
+        data,
+      });
+    });
+
+    this._pingService = new HeartbeatService({
+      heartbeat: async () => {
+        if (ws.readyState !== WebSocket.OPEN) {
+          logger.debug("[AuthenticatedSocketConnection] Websocket not open, skipping ping");
+          return;
+        }
+
+        logger.debug("[AuthenticatedSocketConnection] Sending ping", {
```

Comment:
Same with this one. If you want to keep these maybe add a `logger.dev` or something that only logs when NODE_ENV=development

Reply:
Think I'll just comment this out for now